### PR TITLE
[Feature]: Asset: Expose Embedded Meta Info

### PR DESCRIPTION
**Pull request by bwaidelich**
Extends the `AssetType` by a new field `embeddedMetaInfo` that allows to fetch the asset's Embedded Meta Info as array:

```graphql
query {
  getAsset(id: 123) {
    embeddedMetaInfo {
      name
      value
    }
  }
}

```

Result (exemplarily):

```json
{
  "data": {
    "getAsset": {
      "embeddedMetaInfo": [
        {
          "name": "FileSize",
          "value": "131 kB"
        },
        {
          "name": "FileModifyDate",
          "value": "2019:08:23 15:04:11+02:00"
        },
        {
          "name": "FileAccessDate",
          "value": "2019:08:23 15:04:11+02:00"
        },
        {
          "name": "FileInodeChangeDate",
          "value": "2019:08:23 15:04:11+02:00"
        },
        {
          "name": "FilePermissions",
          "value": "rwxrwxr-x"
        },
        {
          "name": "FileType",
          "value": "JPEG"
        }
      ]
    }
  }
}
```

This will resolve https://github.com/pimcore/pimcore/discussions/8395

### CODE REVIEW

**Comment by mcop1** [comment]
@bwaidelich Thanks for your contribution, looks good to me. Since you have already posted an example with your pr, could you please add it to the documentation too? Maybe somewhere here:
https://pimcore.com/docs/data-hub/current/GraphQL/Query/Query_Samples/Sample_Asset_Metadata.html

File: doc/10_GraphQL/04_Query/11_Query_Samples/12_Sample_Asset_Metadata.md [hunk 1]
```diff
@@ -80,4 +80,99 @@ Get the custom asset metadata for language `de`
...
+      name
+      value
+    }
+  }
+}
+```
+
+### Response
+
+```graphql
```
Comment (bwaidelich) [inline doc/10_GraphQL/04_Query/11_Query_Samples/12_Sample_Asset_Metadata.md:104]: I used "graphql" here to be in sync with the rest of the document, but actually the response is `json` and you might want to consider changing that for a proper highlighting :)

**Comment by bwaidelich** [comment]
@mcop1 done, with one inline comment regarding the docs response format

File: doc/10_GraphQL/04_Query/11_Query_Samples/12_Sample_Asset_Metadata.md (hunk 1 above)
Comment (mcop1) [reply doc/10_GraphQL/04_Query/11_Query_Samples/12_Sample_Asset_Metadata.md:104]: Good point, thanks, it is fine for now, but I created a follow up ticket
https://github.com/pimcore/data-hub/issues/691

**Comment by mcop1** [comment]
@bwaidelich sorry to bother you again, could you please merge the updated 1.x branch into your pr again? This should fix the problems with the failing test.

**Comment by bwaidelich** [comment]
no problem, done

**Comment by mcop1** [comment]
> no problem, done

Perfect, thanks 👍

**Pull request merged**